index.bs

```diff
@@ -355,32 +355,46 @@ User agents may use a [=new information about the user's intent|number of criter
 to grant access to the [=sensor readings|readings=].
 Access may be granted without prompting the user.
 
-<h3 id="mitigation-strategies-case-per-case">Mitigation strategies applied on a case per case basis</h3>
+<h3 id="mitigation-strategies-case-by-case">Mitigation strategies applied on a case by case basis</h3>
 
-Each [=sensor type=] will need to be assessed individually
-taking in account the use cases it enables
+Each [=sensor type=] will need to be assessed individually,
+taking into account the use cases it enables
 and its particular threat profile.
 While some of the below mitigation strategies
 are effective for certain sensors,
-they might also hinder or altogehter prevent certain use cases.
+they might also hinder or altogether prevent certain use cases.
+
+Note: These mitigation strategies can be apllied constantly or temporarily,
+for example when the user is carrying out specific actions,
+when other APIs which are known to amplify the level of the threat are in use,
+etc.
 
 
 <h4 id="limit-max-freqency" dfn>Limit maximum polling frequency</h4>
 
 User agents may mitigate certain threats by
 limiting the maximum polling frequency.
-What maximum polling frequency to choose depends on the [=sensor type=],
+What upper limit to choose depends on the [=sensor type=],
 the kind of threats the user agent is trying to protect against,
-the resources of the attacker, etc.
+the expected resources of the attacker, etc.
 
 Limiting the maximum polling frequency prevents use cases
 which rely on low latency or high data density.
 
 
+<h4 id="stopping-sensor" dfn>Stopping the sensor altogether</h4>
+
+This is ovbiously a last-resort solution,
+but it can be extremely effective if it's temporal,
+for example to prevent password skimming attempts
+when the user is entering credentials on a different origin ([[rfc6454]])
```

Comment:
github wont let me comment on the line below so here goes: or IN a different application